python/pyspark/sql/connect/functions/builtin.py

```diff
@@ -76,15 +76,6 @@
     from pyspark.sql.connect.udtf import UserDefinedTableFunction
 
 
-def _to_col_with_plan_id(col: str, plan_id: Optional[int]) -> Column:
```

Comment:
delete this helper function due to the behavior difference between

https://github.com/apache/spark/blob/d2f572428be5346dfa412f6588e72686429ddc71/sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala#L1452-L1461

https://github.com/apache/spark/blob/0a791993be7b6f4b843887403460ef9aebe3daf9/sql/core/src/main/scala/org/apache/spark/sql/Column.scala#L154-L162